### PR TITLE
Update stylelint peer dependency to include stylelint 15

### PR DESCRIPTION
**Pull request by alex-e-leon**
Bumps peer dependency to fix: https://github.com/csstools/stylelint-value-no-unknown-custom-properties/issues/32

Some quick testing suggests that this plugin works fine with stylelint 15

### CODE REVIEW

**Comment by alex-e-leon** [comment]
For anyone waiting on this, feel free to use my published fork:
`@alex-e-leon/stylelint-value-no-unknown-custom-properties": "^4.0.2"`

**Comment by alex-e-leon** [comment]
Hey @Antonio-Laguna - this should be an easy PR to approve - I've been running this branch for the last few weeks and can confidently say that this plugin still works fine with stylelint 15.

Any chance you can get this merged in?

**Comment by jameschensmith** [comment]
I'm not able to approve (don't have write access), but LGTM 👍 

Also, just a note: a very basic implementation of this has been [added to stylelint](https://github.com/stylelint/stylelint/tree/dca4e789a8c15da76267d322fd4541513fcb6547/lib/rules/no-unknown-custom-properties) (you can view the details in the [initial MR](https://github.com/stylelint/stylelint/pull/6731)). [This tracking issue](https://github.com/stylelint/stylelint/issues/4088) would probably need to be close for the built-in rule to completely replace this rule.

**Comment by alex-e-leon** [comment]
Thanks @jameschensmith : )
I know that at least for my usecase, and probably most people who use css-modules, not being able to track definitions in other files would be a dealbreaker.

**Review by Antonio-Laguna — APPROVED** [review]

**Comment by Antonio-Laguna** [comment]
Thanks @alex-e-leon ! Apologies this took me a while!

**Pull request merged**
